package.json

```diff
@@ -52,13 +52,13 @@
 			"title": "vscode-erg",
 			"properties": {
 				"vscode-erg.ergpath.path": {
```

Comment:
Looking at other language extensions, it does not seem common to add the `vscode-` prefix, so you may want to remove it